content/concepts/consensus-network/consensus-protections.md

```diff
@@ -0,0 +1,67 @@
+- Digital signatures from Ripple employees on all releases and official software packages.
+- Regularly-commissioned professional reviews for security vulnerabilities and insecurities.
+- A [bug bounty program](https://ripple.com/bug-bounty/) that rewards security researchers who responsibly disclose vulnerabilities.
+
+
+## Sybil Attacks
+
+A _[sybil attack](https://en.wikipedia.org/wiki/Sybil_attack)_ is an attempt to take control of a network using a large number of fake identities. In the XRP Ledger, a sybil attack would take the form of running a large number of validators, then convincing others to trust those validators. This sort of attack is theoretically possible, but would be very difficult to accomplish because human intervention is necessary for validators to become trusted.
+
+No matter how many validating servers a would-be attacker runs, those servers have no say on what the existing participants consider validated unless those participants choose to trust the attacker's validators. Other servers only listen to the validators they are configured to trust, either through a validator list or explicit configuration. (See [validator overlap requirements](#validator-overlap-requirements) for a summary of how the default validator list works.)
+
+This trust does not happen automatically, so performing a successful sybil attack would involve the difficult work of convincing targeted humans and businesses to reconfigure their XRP Ledger servers to trust the attacker's validators. Even in the case that one individual entity was fooled into doing so, this would have a minimal impact on others who do not change their configurations.
+
+
+## 51% Attack
+
+A "51% attack" is an attack on a blockchain system where one party controls more than 50% of all mining or voting power. (Technically, the attack is slightly misnamed because _any_ amount over 50% is enough.) The XRP Ledger is not vulnerable to a 51% attack because it does not use mining in its consensus mechanism. The next closest analogue for the XRP Ledger is a [sybil attack](#sybil-attacks), which would also be difficult to accomplish.
+
+
+## Validator Overlap Requirements
+
+For all participants in the XRP Ledger to agree on what they consider validated, they must start by choosing a set of trusted validators that are fairly similar to the sets chosen by everyone else. In the worst case, less than about 90% overlap could cause some participants to diverge from each other. For that reason, Ripple publishes a signed list of recommended validators, including trustworthy and well-maintained servers run by the company, industry, and community.
+
+By default, servers in the network are configured with the URL of Ripple's validator list site, and a cryptographic key that can be used to verify the integrity of lists published to that site. In case the site goes down, the signed updates to the list can also be relayed directly among servers in the XRP Ledger's peer-to-peer network.
```

Comment:
I have to admit that I don't quite understand the first part of the first sentence. What does it mean that the servers in the network are configured with the URL of Ripple's validator list site? Ah, does it mean that by default, they are configured to use the list of validators on Ripple's validator list site? I think I get it. For folks who haven't configured a server, perhaps this may not be all that clear? Maybe consider editing it to something like this?

> By default, servers in the network are configured to trust validators listed on Ripple's validator list site,...

Reply:
They are literally "configured with a URL and a cryptographic key", as in, the configuration contains:

- a URL: https://github.com/ripple/rippled/blob/develop/cfg/validators-example.txt#L58
- a public key: https://github.com/ripple/rippled/blob/develop/cfg/validators-example.txt#L61

The URL is the primary source of the recommended list of validators. Servers that are configured with a validator list site pull this list and verify that the list is signed with a key matching the public key they are configured with. If it matches, those servers set their UNL to match the list: that is, they trust all validators on that list and no others.

If the URL is unavailable, servers can relay the (most recently seen) contents of the site directly to one another. Because the validator list site's contents are signed and versioned, a `rippled` server can confirm whether the list its peers sent it is legit and more recent than any lists it already knows about.

Reply:
Here's what I came up with:

> By default, XRP Ledger servers are configured to use a validator list site run by Ripple. The site provides a list of recommended validators, which Ripple updates periodically. Servers configured this way trust all validators in the latest version of the list, which ensures 100% overlap with others also using the same list. The default configuration includes a public key that verifies the authenticity of the site's contents. In case the site goes down, servers in the XRP Ledger's peer-to-peer network can directly relay the signed updates to the list among themselves.